### DISCUSSION TITLE
DIVE mode settings backport to GoPro 11?

### DISCUSSION THREAD

**Discussion by shokwaav** [gopro/labs#1200] [Feature Requests]
Is it possible for the new colour balance and hypersmooth settings to be backported to the GoPro 11 labs firmware?

**Comment by dnewman-gpsw** [comment]
Maybe not, DIVE HS is complicated, they seems to be too many changes between camera generations.  The more recent white balance changes might be easier to backport.  Generally Labs backports to not get many of the newer features, mainly the improvements to the Labs camera control language.

**Reply by shokwaav** [reply]
Thank you, I'll look into upgrading then.

**Reply by shokwaav** [reply]
Does DIVE HS work when wet lenses are added? Or is hypersmooth calibrated based on the FOV of the lens specified?

For example these [backscatter ones. ](https://www.backscatter.com/Backscatter-GoPro-Action-Cam-Sharp-Wide-Lens-Pro)

**Reply by dnewman-gpsw** [reply]
If you are shooting with a dome, or lens attachment with a significant air gap, then you would want HS prescription for air, not for water.  The DIVE mode corrects for the water distortion as water hits the front flat lens cover.
